<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalMain">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalMain">
        <item>
         <widget class="QStackedWidget" name="rightStack">
          <widget class="QWidget" name="page">
           <widget class="QTableWidget" name="rxtable">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>40</y>
              <width>1011</width>
              <height>371</height>
             </rect>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <column/>
            <column/>
           </widget>
           <widget class="QPushButton" name="btnScroll">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>2</y>
              <width>41</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>S</string>
            </property>
           </widget>
           <widget class="QLCDNumber" name="lcddcin">
            <property name="geometry">
             <rect>
              <x>343</x>
              <y>0</y>
              <width>121</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 255, 0);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="digitCount">
             <number>4</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
           <widget class="QLCDNumber" name="lcddcout">
            <property name="geometry">
             <rect>
              <x>500</x>
              <y>0</y>
              <width>121</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 255, 0);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="digitCount">
             <number>4</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
           <widget class="QLCDNumber" name="lcddcamp">
            <property name="geometry">
             <rect>
              <x>640</x>
              <y>0</y>
              <width>121</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 255, 0);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="digitCount">
             <number>4</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_2"/>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOps">
    <property name="title">
     <string>Ops</string>
    </property>
    <widget class="QMenu" name="menuPort">
     <property name="title">
      <string>Port</string>
     </property>
     <addaction name="actionCom"/>
    </widget>
    <addaction name="menuPort"/>
   </widget>
   <addaction name="menuOps"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>COM</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
